### ISSUE TITLE
[BUG] - Can't add new command when connection is in closed state

### ISSUE THREAD

**Issue by CloudTheWolf** [CloudTheWolf/legacy-mdt-issue-tracker#8]
```
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

    at async file:///usr/src/app/apis/announcements.js:135:31 {

  code: undefined,

  errno: undefined,

  sql: undefined,

  sqlState: undefined,

  sqlMessage: undefined

}

Error getting characters: Error: Can't add new command when connection is in closed state

    at PromiseConnection.query (/usr/src/app/node_modules/mysql2/promise.js:94:22)

    at MySql2PreparedQuery.execute (file:///usr/src/app/node_modules/drizzle-orm/mysql2/session.js:50:33)

    at MySqlSelectBase.execute (file:///usr/src/app/node_modules/drizzle-orm/mysql-core/query-builders/select.js:671:27)

    at getCharactersFromDb (file:///usr/src/app/apis/characters.js:34:28)

    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

    at async Promise.all (index 1)

    at async file:///usr/src/app/apis/roster.js:1187:57 {

  code: undefined,

  errno: undefined,

  sql: undefined,

  sqlState: undefined,

  sqlMessage: undefined

}

```

**Comment by CloudTheWolf** [comment]
This is a bit of an odd one as both pd-server-stg.legacyrp.company and pd-server.legacyrp.company use the same version of the server with the same config and DB (using different users) but only the Staging instance encounters this issue

**Comment by CloudTheWolf** [comment]
This looks to now be solved with check-in [119ea3a](https://github.com/CloudTheWolf/v2-mdt-server/commit/119ea3ac916dac2a5f2dc61bd5d03bc4af6addca)
